modeling-cmds/src/datetime.rs

```diff
@@ -0,0 +1,42 @@
+use kittycad_execution_plan_traits::{MemoryError, NumericPrimitive, Primitive, Value};
+
+/// A wrapper for chrono types, since we need to impl Value for them.
```

Comment:
A unit test showing that you can run this type through `into_parts()` and then back through `from_parts()` and get the original data back would be helpful